repos.md

```diff
@@ -190,7 +190,10 @@
 - http4s/http4s-jdk-http-client
 - http4s/jawn-fs2
 - http4s/rho
-- hntd187/spotify
```

Comment:
Looks like this one was deleted accidentally by a merge? @kailuowang @hntd187

Reply:
Good catch, @ghostdogpr! I added it again in https://github.com/fthomas/scala-steward/commit/896fd9e8775bc305be07049df8d2ac1b857542de